src/SmartComponents/HybridInventoryTabs/ConventionalSystems/RecommendationSystems.js

```diff
@@ -30,6 +36,8 @@ const ConventionalSystems = ({ rule, afterDisableFn, handleModalToggle }) => {
       permsExport={envContext.isExportEnabled}
       exportTable="systems"
       showTags={true}
+      axios={axios}
+      IopRemediationModal={props.props.IopRemediationModal}
```

Comment:
This causes an error when running with stage. Should it maybe just be `props.IopRemediationModal`?